github-pullrequest-plugin/src/main/java/org/jenkinsci/plugins/github/pullrequest/events/impl/GitHubPRCommentEvent.java

```diff
@@ -44,15 +44,16 @@ public GitHubPRCommentEvent(String comment) {
     @Override
     public GitHubPRCause check(GitHubPRTrigger gitHubPRTrigger, GHPullRequest remotePR,
                                @CheckForNull GitHubPRPullRequest localPR, TaskListener listener) {
-        if (isNull(localPR) || isNull(localPR.getLastCommentCreatedAt())) {
```

Comment:
i mistake. So if there were no comments before (initially) and on the next check they appear this place shouldn't skip it?